setup.cfg

```diff
@@ -23,6 +23,7 @@ classifiers =
 
 [options]
 py_modules = tokenize_rt
+install_requires = typing; python_version=='2.7'
```

Comment:
I think this is supposed to be `python_version="2.7"`

Reply:
Yea saw the CI error after the fact 🤦‍♂

Reply:
I think this was right but I needed a newline?